src/dotnet/Fable.Client.Browser/Fable.Client.Browser.fsproj

```diff
@@ -30,8 +28,7 @@
     <Compile Include="../Fable.Compiler/FSharp2Fable.Util.fs"/>
     <Compile Include="../Fable.Compiler/FSharp2Fable.fs"/>
     <Compile Include="../Fable.Compiler/Fable2Babel.fs"/>
-
-    <Compile Include="../Fable.Tools/State.fs"/>
+    <Compile Include="../Fable.Compiler/State.fs"/>
```

Comment:
I think there's a way to set the configuration of the referenced project and then use that configuration as a condition to set the FABLE_COMPILER flag. At least in the old project format, not sure if it's the same (or maybe there's an easier way) in the new project. Summoning @enricosada the wizard :wink: